osl/interface_linux.go

```diff
@@ -415,6 +425,28 @@ func setInterfaceRoutes(nlh *netlink.Handle, iface netlink.Link, i *nwIface) err
 	return nil
 }
 
+func enableDisableIPv6(nlh *netlink.Handle, iface netlink.Link, i *nwIface) error {
+	var (
+		err    error
+		action = "disable"
+		value  = byte('1')
+		path   = fmt.Sprintf("/proc/sys/net/ipv6/conf/%s/disable_ipv6", i.DstName())
+	)
+
+	if i.ipv6Enabled {
+		action = "enable"
+		value = byte('0')
+	}
+
+	i.ns.InvokeFunc(func() {
+		if err = ioutil.WriteFile(path, []byte{value, '\n'}, 0644); err != nil {
```

Comment:
Since we are calling `ioutil.WriteFile` these are potential namespace corruption paths. We might have to reexec them.